stub-environment/roles/BCPC-Hadoop-Head.json

```diff
@@ -17,6 +17,8 @@
     "recipe[bcpc::graphite]",
     "recipe[bcpc::diamond]",
     "recipe[bcpc::zabbix-head]",
+    "recipe[bcpc::ssh_cert_deploy]",
```

Comment:
Rather than adding `ssh_cert_deploy` here, maybe we should add it to Basic?

Reply:
Yes, that's better.